pkg/apis/core/types.go

```diff
@@ -2608,6 +2608,12 @@ type PodSecurityContext struct {
 	// +k8s:conversion-gen=false
 	// +optional
 	HostIPC bool
+	// Use host's user namespace for the pods.
+	// This field is alpha-level and can be set to false on a node only if user-namespace remapping feature is enabled at runtime and kubelet.
+	// Optional: Defaults to true, if feature-gate is enabled
```

Comment:
I thought we discussed this staying nil by default, meaning runtime-defined behavior

Reply:
If you mean we should keep default as nil (irrespective of feature gate on or off), because CRI is passing "NODE" as user namespace option to runtime by default, overall effect will be same. Even keeping nil, will provide host usernamespace in container because of CRI defaulting.